compiler/luci/pass/src/RemoveUnnecessaryTransposeNetPass.cpp

```diff
@@ -24,38 +24,15 @@
 namespace
 {
 
-class TaggedShapeAnalyzer final
+struct Dim final
 {
-public:
-  template <loco::DataType DType>
-  bool can_remove_transposes(const luci::CircleTranspose *f_tr, const luci::CircleReshape *m_rs,
-                             const luci::CircleTranspose *b_tr);
-
-private:
-  void init_shape_with_tag(const luci::CircleNode *);
-
-  template <loco::DataType PermType> void analyze_transpose(const luci::CircleTranspose *);
-
-  template <loco::DataType ShapeType> bool analyze_reshape(const luci::CircleReshape *);
-
-  bool verify_tag() const;
-
-  struct Dim final
-  {
-    int32_t value;
-    std::vector<uint8_t> tags;
-  };
-
-  const uint8_t START_TAG = 0;
-
-  using Shape = std::vector<Dim>;
-  Shape _shape;
-
-  int32_t flatsize(const Shape &shape) const;
-  bool inference_incomplete_shape(const Shape &src, Shape &dst);
+  int32_t value;
+  std::vector<uint8_t> tags;
 };
 
-int32_t TaggedShapeAnalyzer::flatsize(const Shape &shape) const
+using Shape = std::vector<Dim>;
```

Comment:
As there exist other `Shape` and also declares here too, how about use `TagsShape` ? 
and also `TagsDim` instead of `Dim` ?

Reply:
Ah, That's a clever idea :)
Thanks for the suggestion. I'll apply it.